.gitattributes

```diff
@@ -0,0 +1,4 @@
+manifest.go -diff -merge
```

Comment:
Can we specify the paths to these files so as to avoid matching on potentially other files that get added to the repository with the same names but different purposes?